src/components/DpFlyout/DpFlyout.vue

```diff
@@ -67,6 +68,13 @@ export default {
       default: true
     },
 
+    mode: {
```

Comment:
Nitpick: in many other cases we use the term "variant" for this kind of variation.

Reply:
Good point, changed in 7fe095ea9732a4f6afb296e69b807a2c1f46bb77